soroban-env-host/src/fees.rs

```diff
@@ -13,6 +13,9 @@ pub const TTL_ENTRY_SIZE: u32 = 48;
 const INSTRUCTIONS_INCREMENT: i64 = 10000;
 const DATA_SIZE_1KB_INCREMENT: i64 = 1024;
 
+// minimum effective write fee per 1KB
+pub const MINIMUM_WRITE_FEE_PER_1KB: i64 = 1000;
```

Comment:
I wonder, why not 1 or even 0? This adds some non-trivial logic to already complex fee logic... I guess it's too late to change this, just curious as to whether this was intentional.

Reply:
yes, this guardrail matches `InitialSorobanNetworkConfig::BUCKET_LIST_FEE_1KB_BUCKET_LIST_LOW` Soroban default that is supposed to be the "lowest sane value" for pubnet.
We can always rewrite the formula later as this should really never trigger in pubnet.